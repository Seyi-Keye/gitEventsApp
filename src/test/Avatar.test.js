import React from "react";
import Avatar from "../Avatar";
import renderer from "react-test-renderer";

describe("Avatar component", () => {
  let wrapper = renderer.create(<Avatar />);
  it("renders succesfully", () => {
    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  it("renders without props", () => {
    const testInstace = wrapper.root;
    expect(testInstace.props).toEqual({});
  });
});

describe("Avatar component", () => {
  let wrapper = renderer.create(
    <Avatar picture="http://svgur.com/i/65U.svg" />
  );

  it("renders succesfully with props", () => {
    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  it("renders with props", () => {
    const testInstace = wrapper.root;
    expect(testInstace.props).toEqual({
      picture: "http://svgur.com/i/65U.svg"
    });
  });
});
