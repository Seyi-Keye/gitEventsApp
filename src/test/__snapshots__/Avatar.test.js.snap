exports[`Avatar component renders succesfully 1`] = `
<img
  alt="dafault avatar"
  src="http://svgur.com/i/65U.svg" />
`;

exports[`Avatar component renders succesfully with props 1`] = `
<img
  alt="user avatar"
  src={undefined} />
`;
